nomad/grapl-core.nomad

```diff
@@ -3,10 +3,19 @@ variable "rust_log" {
   description = "Controls the logging behavior of Rust-based services."
 }
 
-variable "container_repository" {
-  type        = string
-  default     = ""
-  description = "The container registry in which we can find Grapl services. Requires a trailing / if not empty string"
+variable "container_images" {
+  type        = map(string)
```

Comment:
You'll note I've made the decision to stop using bespoke variables per-service and instead just pipe in a Map[str, str]. I'm willing to revisit this decision but, honestly, I think this makes things a bit cleaner, and lets us recycle the `var.container_images` pattern across files.